.eslintrc.yml

```diff
@@ -0,0 +1,17 @@
+extends: standard
+
+env:
+  node: true
+  mocha: true
```

Comment:
I'm surprised we have `node: true` and Koa doesn't, does eslint pass without it? Also I'd like it to have a separate `.eslintrc` in the test directory with `env: \n mocha: true` since Koa does something similar to prevent testing globals from showing up in other source files

Reply:
Fixed